app/Services/Beanstalkd.php

```diff
@@ -6,6 +6,7 @@ class Beanstalkd extends BaseService
 {
     protected $organization = 'schickling';
     protected $imageName = 'beanstalkd';
+    protected static $category = 'cache';
```

Comment:
Can we make all of these constants on a `Category` class so we can ensure they're consistent? So this line would instead be `protected static $category = Category::CACHE` or something similar

Reply:
Yes also for me is better. In my private projects, I usually create a constant class to manage this strings.
So I created an abstract class Category placed in Services folder. Super clean.